tests/test_recipes.py

```diff
@@ -129,11 +110,34 @@ def test_prune_recipe(recipe_fixture, execute_recipe, nkeep):
 @pytest.mark.parametrize("cache_inputs", [True, False])
 @pytest.mark.parametrize("copy_input_to_local_file", [True, False])
 def test_recipe_caching_copying(
-    netCDFtoZarr_sequential_recipe, execute_recipe, cache_inputs, copy_input_to_local_file
+    netCDFtoZarr_recipe, execute_recipe, cache_inputs, copy_input_to_local_file
 ):
-    """The basic recipe test. Use this as a template for other tests."""
+    """Test that caching and copying to local file work."""
+
+    RecipeClass, file_pattern, kwargs, ds_expected, target = netCDFtoZarr_recipe
+
+    if not cache_inputs:
+        kwargs.pop("input_cache")  # make sure recipe doesn't require input_cache
+    rec = RecipeClass(
+        file_pattern,
+        **kwargs,
+        cache_inputs=cache_inputs,
+        copy_input_to_local_file=copy_input_to_local_file
+    )
+    execute_recipe(rec)
+    ds_actual = xr.open_zarr(target.get_mapper()).load()
+    xr.testing.assert_identical(ds_actual, ds_expected)
+
+
+@pytest.mark.parametrize("cache_inputs", [True, False])
+@pytest.mark.parametrize("copy_input_to_local_file", [True, False])
+def test_recipe_http_caching_copying(
+    netCDFtoZarr_http_recipe, execute_recipe, cache_inputs, copy_input_to_local_file
+):
```

Comment:
This test seems nearly identical to the one above. Could we use a `lazy_fixture` to combine `netCDFtoZarr_recipe` and `netCDFtoZarr_http_recipe` into a single fixture for a single `test_recipe_caching_copying`?

Reply:
👍 https://github.com/pangeo-forge/pangeo-forge-recipes/pull/167/commits/72da50fba4e023bbe170bf9e52ac86211c80934a